modules/builds-docker-credentials-private-registries.adoc

```diff
@@ -115,3 +116,18 @@ This example uses `pullSecret` in a Source build, but it is also applicable
 in Docker and Custom builds.
 ====
 endif::[]
+
+. You can also link the pull secret to the service account used by the build instead of specifying
+the `pullSecret` field. By default builds use the *builder* service account. The pull secret is
+automatically added to the build if the secret contains a credential that matches the repository
+hosting the build's input image.
++
+----
+$ oc secrets link builder dockerhub
+----
+
+[NOTE]
+====
+You must specify a `from` image in the `BuildConfig` spec to take advantage of this feature.
+Docker strategy builds generated by `oc new-build` or `oc new-app` may not do this in some situations.
+====
```

Comment:
@bparees @gabemontero Updated this note per your feedback. PTAL.

Reply:
note looks good